tests/unit/test_routes/test_resource_crud.py

```diff
@@ -0,0 +1,552 @@
+from datetime import datetime
```

Comment:
This file is still too big. We definitely need to break it up more. I think that the `update` stuff can go in one file, and the `create` stuff should definitely be its own file.